src/p2p/net_node.inl

```diff
@@ -2364,7 +2364,7 @@ namespace nodetool
       address = epee::net_utils::network_address{epee::net_utils::ipv6_network_address(ipv6_addr, node_data.my_port)};
     }
     peerid_type pr = node_data.peer_id;
-    bool r = zone.m_net_server.connect_async(ip, port, zone.m_config.m_net_config.ping_connection_timeout, [cb, /*context,*/ address, pr, this](
+    bool r = zone.m_net_server.connect_async(ip, port, zone.m_config.m_net_config.ping_connection_timeout, [cb = std::forward<t_callback>(cb), /*context,*/ address, pr, this](
```

Comment:
If `cb` is an r-value reference, won't this move `cb` into the lambda, making the move down in `[this, pr, cb = std::move(cb), ping_context = std::move(ping_context), address = std::move(address)]` invalid?

Reply:
No, but I think I need to add the `mutable` keyword here because `cb` should be implicit `const`, forcing a silent copy at the `std::move`.

Reply:
Oh yeah I forgot that lambda captures are `const` by default. Thanks for the reminder